docs/api.rst

```diff
@@ -0,0 +1,32 @@
+Astromon API
+============
+
+db
```

Comment:
Every function that is documented here in the "public API" should have complete docstrings with all the parameters documented. You can choose to take functions out of the public API by defining `_all__` and not including them.